schedulingApp/acceptance-tests/test_assignUser.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.models import User
+from django.test import TestCase, Client
+from schedulingApp.models import Course, Profile, CourseToAssignedUserEntry
+
+class TestAssignUserToCourse(TestCase):
+    monkey = None
+
+    def setUp(self):
+        self.monkey = Client()
+        testUser = User.objects.create_user(username="user", password="pass")
+        testUser.save()
+        profile = Profile.objects.get(user=testUser)
+        profile.permission = Profile.ADMIN
+        profile.save()
+        self.monkey.force_login(testUser)
+
+        self.course = Course(title='CS361', semester='FA22')
+        self.course.save()
+        self.taUser = User.objects.create_user(username="test", password="TA", email="testTA@uwm.edu")
```

Comment:
I would maybe make this user a Professor, as this is the primary type of user that needs to be assigned to a course? Not anything to stop approval over, though

Reply:
I can do one for TA and one for Professor, though I dont believe it matters